core/integration/steps/parties_place_the_following_orders.go

```diff
@@ -63,13 +65,17 @@ func PartiesPlaceTheFollowingOrdersWithTicks(exec Execution, time *stubs.TimeStu
 		}
 		// make it look like we start a new block
 		epochService.OnBlockEnd(context.Background())
+		// indicate block has ended, so we MTM if needed
+		exec.BlockEnd(context.Background())
```

Comment:
why every order on it's own block?

Reply:
and why is it even necessary vs the user moving ahead the time / number of blocks

Reply:
Because the step here is `PartiesPlaceTheFollowingOrdersWithTicks`. This is was a relatively simple way to come as close as possible to the old behaviour (where MTM was performed after each trade, so by placing each order in its own block, any order that uncrosses can trigger MTM)